docs/csharp/language-reference/keywords/file.md

```diff
@@ -21,9 +20,9 @@ file class HiddenWidget
 }
 ```
 
-Any types nested within a file-based type are also only visible within the file in which it's declared. Other types in an assembly may use the same name as a file-scoped type. Because the file-scoped type is visible only in the file where it's declared, these types don't create a naming collision.
+Any types nested within a file-scoped type are also only visible within the file in which it's declared. Other types in an assembly may use the same name as a file-scoped type. Because the file-scoped type is visible only in the file where it's declared, these types don't create a naming collision.
```

Comment:
```suggestion
Any types nested within a file-local type are also only visible within the file in which it's declared. Other types in an assembly may use the same name as a file-local type. Because the file-local type is visible only in the file where it's declared, these types don't create a naming collision.
```